src/lib/components/CopyCode/index.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @fileoverview Element that renders copyable code.
+ */
+
+import {BaseElement} from "../BaseElement";
+
+/**
+ * Renders code block that can easily be copied.
+ *
+ * @extends {BaseElement}
+ * @final
+ */
+class CopyCode extends BaseElement {
+  static get properties() {
+    return {
+      code: {type: String},
+    };
+  }
+
+  constructor() {
+    super();
+  }
+
+  connectedCallback() {
```

Comment:
Suggest also adding a `disconnectedCallback` to remove any event listeners and clean up after the element.

Reply:
I was under the impression that event listeners belonging to a custom element, or elements that belong to a custom element are [removed on the elements destruction](https://stackoverflow.com/a/35108388)? I'm assuming this would have only been applicable if the element was using the shadow dom?

Reply:
I think you're correct that the handler will be automatically deleted. Previously Sam and I were encouraging one another to add `disconnectedCallback` to make sure we disconnected elements from the `store`. I think it was muscle memory that encouraged me to leave my previous comment. In this instance it's probably fine to remove the `disconnectedCallback` if you'd like to shave some bytes.